frontend/src/static/js/v_summary.js

```diff
@@ -146,13 +146,20 @@ window.vSummary = {
       if (this.tmpFilterSinceDate && this.tmpFilterSinceDate >= this.minDate) {
         this.filterSinceDate = this.tmpFilterSinceDate;
         this.getFiltered();
+      } else if (!this.tmpFilterSinceDate) { // If user clears the since date field
+        this.filterSinceDate = this.minDate;
+        this.tmpFilterSinceDate = this.filterSinceDate;
+        this.getFiltered();
```

Comment:
Normally when we [comment](https://github.com/reposense/RepoSense/pull/700#discussion_r289009357), it applies to all the code. So here as well.

Reply:
My bad, missed that out as well..